neurom/tests/test_viewer.py

```diff
@@ -52,6 +53,12 @@ def test_draw_neuron():
     common.plt.close('all')
 
 
+def test_draw_filter_neurite():
+    for mode in ['2d', '3d']:
+        viewer.draw(nrn, mode=mode, neurite_type=NeuriteType.basal_dendrite)
```

Comment:
check some bounding boxes or something, to make sure that you're actually plotting only the basal_dendrite